### PR TITLE
Reuse

### PR DESCRIPTION
**Pull request by NSfsfe**
Hello,

By way of introduction, I am Niharika Singhal from the Free Software Foundation Europe, and the REUSE Booster program. Thank you for contacting us and we are happy to help you out with the implementation of REUSE. After a check on your repository, I would like to propose some updates regarding your copyright and licensing information. The REUSE specifications intend to make licensing easier with best practices to display legal information through comment headers on source files that can be easily human and machine readable. You can check this screencast:
https://download.fsfe.org/videos/reuse/screencasts/reuse-tool.gif

Please keep in mind that this merge request is intended to be a practical example of how REUSE looks like.

I would also like to make some remarks to be considered to make your full project REUSE compliant.

Feel completely free to adopt these specifications by merging this pull request.
**REUSE Features:**

SPDX copyright and license comment headers for all relevant files (please be aware that in the current state of the repository, I was able to add headers to all the directories. However, some files might miss this feature depending on the type of the file. If you are interested in implementing REUSE specification, I hope this merge request can help you to do so.
**LICENSES directory in the root of the project with the licenses used on the repository:**

I included in this directory, the files that contained the license for your project (BSD 3 Clause) text. I also notice that the LICENSES directory contains the license CC-BY-SA 3.0, however the same has not been applied to any file in the project. If some pieces of your project use a different license, please add their license text to the LICENSES directory.
**Files missing copyright and licensing information:**

I noticed that your project does not have the copyright and license information displayed in a comment header. According to the REUSE specifications, the comment headers MUST contain the SPDX Tag. For this, we recommend to update the headers, or at least the SPDX-License-Identifier as in this practical example.

Please also double check if the personal information in the headers is correct and consistent, in the case of several copyright holders please update that information in the headers.

In case you want to license certain files under a different license, special attention should also be paid to that aspect and such files should contain the appropriate SPDX tag.

Please note that I used the REUSE helper tool to add the headers, so the tool automatically deleted the existing header (if any) and changed it to the SPDX format.

The REUSE helper tool also detected that the file contained in: libraries/class-star/patch.lisp contains a public domain and hence, the project does not show to be completely REUSE compliant.

**#Image files:**
Copyright and Licensing Information MAY be associated with a file through a DEP5 file that you will find in the .reuse folder in the root of the project. The intended use case of this method is large directories where including a comment header in each file (or in .license companion files) is impossible or undesirable.I added the BSD-3-Clause license to all of the image files. However, you can license other image files under licenses as well and include the license in the LICENSES directory.
We can also do a follow up of this PR and further REUSE specification with a call. Feel free to reach me out.

Hope that helps and thank you very much for the amazing job!

Niharika Singhal - Trainee
Free Software Foundation Europe
Schönhauser Allee 6/7, 10119 Berlin, Germany | t +49-30-27595290
Registered at Amtsgericht Hamburg, VR 17030 | https://fsfe.org/support

### CODE REVIEW

File: source/web-extensions-callbacks.lisp [hunk 1]
```diff
@@ -1,5 +1,14 @@
+<<<<<<< HEAD
+# SPDX-FileCopyrightText: 2017 John Mercouris <john@atlas.engineer>
+# SPDX-FileCopyrightText: 2017 Atlas Engineer LLC
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Atlas Engineer LLC
+=======
```
Comment (Ambrevar) [inline source/web-extensions-callbacks.lisp:8]: Looks like there was a merge issue here :p

File: source/mode/summarize.lisp [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            (serapeum:string-join
+             (map 'list (lambda (e) (plump:text e))
+                  (clss:select "p" (document-model buffer)))
+             " ")))
+      (spinneret:with-html-string
+        (:style (style output))
+        (:h1 "Summary for: " (title buffer))
+        (:ul
+         (loop for point in (analysis:summarize-text contents :summary-length summary-length)
+               collect (:li point)))))))
```
Comment (Ambrevar) [inline source/mode/summarize.lisp:25]: This file should go.

**Comment by Ambrevar** [comment]
Thank you very much for the pull request, you've caught a few issues indeed!  We are going to fix them soon.

`libraries/class-star/patch.lisp` will be removed from the repo very soon.

**Comment by jmercouris** [comment]
Thanks for alerting us to some problems. We will address them! That said, we cannot merge this pull request because it is incompatible with Lisp comments which are prefixed by `;` and not `#`.

**Comment by Ambrevar** [comment]
Looks this this pull request is gone and now has 0 commit.  @NSfsfe any idea what happened?

**Comment by NSfsfe** [comment]
Hi,

I cannot figure out what has happened but I could send you another pull request. This one will incorporate all the modifications you made to the .lisp files, so I am hopeful that it could be merged. :)

**Pull request merged**